htdocs/accountancy/class/accountancyexport.class.php

```diff
@@ -355,21 +355,49 @@ public function exportCogilog($objectLines)
 	 */
 	public function exportCoala($objectLines)
 	{
-		// Coala export
-		$separator = ";";
```

Comment:
Why you remove déclaration of end l'une and separator ?

Reply:
@aspangaro bad paste... to many things in same time :/